Tests/IntegrationTestApp/IntegrationTestAppUITests/LivenessIntegrationUITests.swift

```diff
@@ -0,0 +1,57 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import XCTest
+
+class CreateLivenessSessionUITests: XCTestCase {
+    
+    var app: XCUIApplication?
+    
+    override func setUp() {
+        super.setUp()
+        continueAfterFailure = false
+        app = XCUIApplication()
+        app?.launch()
+      }
+    
+    func testBeginCheckUI() throws {
+        XCTAssertEqual(app!.label, UIConstants.appName)
+        XCTAssert(app!.buttons[UIConstants.primaryButton].exists)
+        XCTAssert(app!.buttons[UIConstants.primaryButton].isEnabled)
+        app!.buttons[UIConstants.primaryButton].tap()
+        Thread.sleep(forTimeInterval: 2)
+        XCTAssert(app!.buttons[UIConstants.BeginCheck.primaryButton].exists)
+        XCTAssertFalse(app!.buttons[UIConstants.primaryButton].exists)
+        let scrollViewsQuery = app!.scrollViews
+        let elementsQuery = scrollViewsQuery.otherElements
+        XCTAssert(elementsQuery.staticTexts[UIConstants.BeginCheck.description].exists)
```

Comment:
I'm getting this error when running the test locally.
Otherwise LGTM.
```
testBeginCheckUI(): XCTAssertTrue failed: throwing "NSInternalInconsistencyException: Invalid query - string identifier 'You will go through a face verification process to prove that you are a real person. Your screen's brightness will temporarily be set to 100% for highest accuracy.' exceeds maximum length of 128 characters. You can work around this limitation by constructing a query with a custom NSPredicate that specifies the property (label, title, value, placeholderValue, or identifier) to match against."
```

Reply:
fixed assertion failing due to the text to query was too long